docs/data_cycling.md

```diff
@@ -0,0 +1,51 @@
+# Data retention policies
+
+## On Perfherder
+
+On a daily basis, Perfherder expires data out of several reasons:
+
+* data provides lower & lower interest as it grows older
+* data accumulates very fast (>1 million new data points are ingested daily)
+* query latency degrades in time, down to intermittent timeouts
+* database is rather limited (in terms of storage capacity & scalability)
+
+To ensure persistence of the most relevant performance data, Perfherder's cycling algorithm takes a more aggressive approach towards the less relevant one. It employs multiple expiring strategies, each one specialized on deleting specific sets of data.
+
+Basically, not all data is deleted in the same way. Some data sets can be kept for longer time than others.
+
+Data targeted for removal includes:
+
+* data points
+* series (AKA performance signatures; they collect data points sharing same characteristis)
+* alerts
+* alert summaries
+
+Generally, the daily cycling starts by removing data points first, using its defined strategies. Then it continues with removing series, alerts & alert summaries using a garbage collection approach.
```

Comment:
In reference to  "using its defined strategies", is there a default that can be configurable based on flags??

Reply:
All expiring strategies are complementary & always run sequentially. Only together are they able to delete more amounts of irrelevant data.
Because of this, there isn't a CLI option to select a particular strategy.

I've tweaked the comment a bit to reflect this.